exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/_metric_exporter/__init__.py

```diff
@@ -29,12 +30,19 @@
     InstrumentationLibraryMetrics,
     ResourceMetrics,
 )
-from opentelemetry.proto.metrics.v1.metrics_pb2 import Metric as PB2Metric
+from opentelemetry.proto.metrics.v1.metrics_pb2 import NumberDataPoint as OTLPNumberDataPoint
+from opentelemetry.proto.metrics.v1.metrics_pb2 import Gauge as OTLPGauge
+from opentelemetry.proto.metrics.v1.metrics_pb2 import Histogram as OTLPHistogram
+from opentelemetry.proto.metrics.v1.metrics_pb2 import Metric as OTLPMetric
+from opentelemetry.proto.metrics.v1.metrics_pb2 import Sum as OTLPSum
```

Comment:
nit Might be cleaner to `from opentelemetry.proto.metrics.v1 import metrics_pb2` and use that module namespace

Reply:
Agreed, cleaned up